### PR TITLE
cos template update for charts 2.2.18

### PR DESCRIPTION
**Pull request by Bhagyashreek8**

### CODE REVIEW

File: config-templates/ibm/ibm-object-storage-plugin/changelog.json [hunk 1]
```diff
@@ -3,6 +3,17 @@
         "2.2": {
             "status": "supported",
             "latest-revision": {
+                "revision": "14",
```
Comment (nkkashyap) [inline config-templates/ibm/ibm-object-storage-plugin/changelog.json:6]: Need to update revision/chart version in the template `ibm-object-storage-plugin/2.2/deployment.yaml`

**Review by nkkashyap — CHANGES_REQUESTED** [review]

File: config-templates/ibm/ibm-object-storage-plugin/changelog.json (hunk 1 above)
Comment (Bhagyashreek8) [reply config-templates/ibm/ibm-object-storage-plugin/changelog.json:6]: Fixed it.

**Review by mssachan — APPROVED** [review]
LGTM

**Comment by mssachan** [comment]
@nkkashyap please re-approve the PR.

**Review by gauravAggrahari — APPROVED** [review]
PR LGTM!

**Review by gauravAggrahari — APPROVED** [review]
PR LGTM!

**Review by gauravAggrhari — APPROVED** [review]
PR LGTM!

**Review by nkkashyap — APPROVED** [review]
lgtm

**Pull request merged**